### DISCUSSION TITLE
Will the online test session be updated?

### DISCUSSION THREAD

**Discussion by Ethan1X** [PaddlePaddle/PaddleSpatial#210] [BaiduKDDCup2022]
The announcement stated that the online test set will be updated on June 20. However, from the current submission results, it seems that the online test set has not been updated. Will it be updated? When will it be updated?

**Comment by AlvinAi96** [comment]
Mark, we got the same questions as you.

**Reply by fersebas** [reply]
Same here

**Reply by luckyyangrun** [reply]
Same here

**Comment by shaido987** [comment]
The second phase test data is available now, with a new leaderboard. 

~~But there looks to be some issues with the packages for now, see the issue I reported here https://github.com/PaddlePaddle/PaddleSpatial/discussions/218.~~ This issue appears fixed already.
